packages/client/components/ErrorBoundary.tsx

```diff
@@ -15,20 +17,31 @@ interface State {
   error?: Error
   errorInfo?: ErrorInfo
   eventId?: string
+  isOldBrowserErr: boolean
 }
 
 class ErrorBoundary extends Component<Props, State> {
   state: State = {
     error: undefined,
     errorInfo: undefined,
-    eventId: undefined
+    eventId: undefined,
+    isOldBrowserErr: false
+  }
+
+  componentDidUpdate() {
```

Comment:
I didn't refactor this component to hooks because there isn't a hooks equivalent to `componentDidCatch`: https://reactjs.org/docs/hooks-faq.html#do-hooks-cover-all-use-cases-for-classes